lib/migration/page_language.rb

```diff
@@ -0,0 +1,27 @@
+module Migration
+  ##
+  # This migration sets the Spotlight::Page locale to the I18n.default_locale.
+  # Needed for migrating Exhibits to internationalization work.
+  class PageLanguage
+    def self.run
+      new.run
+    end
+
+    def initialize; end
+
+    def run
+      migrate_pages
+    end
+
+    private
+
+    def migrate_pages
+      FriendlyId::Slug.where(sluggable_type: 'Spotlight::Page').find_each do |slug|
+        unless /locale:\w+/ =~ slug.scope
+          slug.scope += ",locale:#{I18n.default_locale}"
```

Comment:
Only thing I can think of here is we may want to do something like `slug.scope = [slug.scope, "locale:#{I18n.default_locale}"].join(',')` to guard against any blank column.

I would be **extremely** surprised if any of our scope columns for pages where blank, so this is more of a comment than a change request.